### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/fx-private-relay/commit/2ca1629e9b3ff678217cfccdcb8430a041151228#commitcomment-80568351

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/fx-private-relay/commit/0725106052b0cf4a67b7e3ac9bbf64e19f68bfed

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

This PR fixes #<issue ID>.

How to test:

- [ ] l10n changes have been submitted to the l10n repository, if any. -- They have not.
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] I've added or updated relevant docs in the docs/ directory.
- [ ] All UI revisions follow the [coding standards](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md), and use Protocol tokens where applicable (see `/frontend/src/styles/tokens.scss`).
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: emails/models.py [hunk 1]
```diff
@@ -397,7 +397,7 @@ def is_flagged(self):
         if datetime.now(timezone.utc) > account_premium_feature_resumed:
             # premium feature has been resumed
             return False
-        # user was flagged and the premiume feature pause period is not yet over
+        # user was flagged and the premium feature pause period is not yet over
```
Comment (jsoref) [inline emails/models.py:400]: ?

File: frontend/__mocks__/modules/next__router.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 export const mockNextRouter = {
   useRouter: jest.fn(() => {
     return {
-      pathmame: "/",
+      pathname: "/",
```
Comment (jsoref) [inline frontend/__mocks__/modules/next__router.ts:4]: ??

File: frontend/pendingTranslations.ftl [hunk 3]
```diff
@@ -151,7 +151,7 @@ phone-onboarding-step3-input-placeholder = Enter 6-digit code
 phone-onboarding-step3-button-cta = Confirm my phone number
 phone-onboarding-step3-button-edit = Edit true phone number
 phone-onboarding-step3-button-resend = Resend verification code
-phone-onboarding-step3-error-exipred = Try again—the time limit expired.
+phone-onboarding-step3-error-expired = Try again—the time limit expired.
```
Comment (jsoref) [inline frontend/pendingTranslations.ftl:154]: Presumably changes to this file require changes elsewhere. I'm unlikely to have the resources to do that.

File: frontend/src/pages/accounts/profile.test.tsx [hunk 4]
```diff
@@ -5,7 +5,7 @@ import { axe } from "jest-axe";
 import { mockFluentReact } from "../../../__mocks__/modules/fluent__react";
 import { mockNextRouter } from "../../../__mocks__/modules/next__router";
 import { mockReactGa } from "../../../__mocks__/modules/react-ga";
-import { mockReactIntersectionObsever } from "../../../__mocks__/modules/react-intersection-observer";
+import { mockReactIntersectionObserver } from "../../../__mocks__/modules/react-intersection-observer";
```
Comment (jsoref) [inline frontend/src/pages/accounts/profile.test.tsx:8]: Dangerous?

**Review by jsoref — COMMENTED** [review]
Most changes automatically suggested by Google Sheets. All fault mine

**Comment by groovecoder** [comment]
Hi, thanks for the PR. Since this touches so many files at once, we probably won't merge this until we have planned a full end-to-end manual regression test run. So it could take a while before we get to this.

**Comment by groovecoder** [comment]
I'm gong to close this PR but keep the branch on GitHub so we can get back to it later.

**Pull request closed without merging**